cf-templates/ekscluster.yaml

```diff
@@ -40,7 +42,7 @@ Resources:
       ServiceToken: !GetAtt DeployCloudformationStack.Arn
       cfStackName: "eksDeploy"
       assumeRoleARN: !Ref C9Role
-      templateUrl: !Sub https://${S3BucketName}.s3.amazonaws.com/assets/eks.yaml
+      templateUrl: !Sub https://${S3BucketName}/eks.yaml
```

Comment:
Please correct the templateUrl because we need to make it ready to merge.

Reply:
These changes are necessary to use the bucket managed by the workshop.